spring-integration-core/src/main/java/org/springframework/integration/channel/AbstractMessageChannel.java

```diff
@@ -116,6 +119,16 @@ public boolean isStatsEnabled() {
 		return this.statsEnabled;
 	}
 
+	protected AbstractMessageChannelMetrics getMetrics() {
+		return this.channelMetrics;
+	}
+
+	@Override
+	public void configureMetrics(ConfigurableMetrics metrics) {
```

Comment:
No; I think a global option is sufficient. I don't see a need for more. It is true they could do it with java config, but I seriously doubt they would.